docs/en/ingest-management/fleet/fleet-settings-output-elasticsearch.asciidoc

```diff
@@ -89,6 +89,31 @@ output is set in the <<agent-policy,agent policy>>.
 | Choose one of the menu options to tune your {agent} performance when sending data to an {es} output. You can optimize for throughput, scale, latency, or you can choose a balanced (the default) set of performance specifications. Refer to <<es-output-settings-performance-tuning-settings,Performance tuning settings>> for details about the setting values and their potential impact on performance.
 
 You can also use the <<es-output-settings-yaml-config,Advanced YAML configuration>> field to set custom values. Note that if you adjust any of the performance settings described in the following **Advanced YAML configuration** section, the **Performance tuning** option automatically changes to `Custom` and cannot be changed.
+
+Performance tuning preset values take precedence over any settings that may be defined separately. If you want to change any setting, you need to use the `Custom` **Performance tuning** option and specify the settings in the **Advanced YAML configuration** field.
+
+For example, if you would like to use the balanced preset values except that you prefer a higher compression level, you can do so as follows:
+
+. In {fleet}, open the **Settings** tab.
+. In the **Outputs** section, select **Add output** to create a new output, or select the edit icon to edit an existing output.
+. In the **Add new output** or the **Edit output** flyout, set **Performance tuning** to `Custom`.
+. Refer to the list of <<es-output-settings-performance-tuning-settings,performance tuning preset values>>, and add the settings you prefer into the **Advanced YAML configuration** field. For the `balanced` presets, the yaml configuration would be as shown:
++
+[source,yaml]
+----
+bulk_max_size: 1600
+worker: 1
+queue.mem.events: 3200
+flush.min_events: 1600
+flush.timeout: 10
```

Comment:
These need the `queue.mem.` prefix per https://github.com/elastic/kibana/issues/175892

Reply:
Thanks @kpollich. I've pushed the change here and at line 231 below.

I'm a bit confused though. Would you know if these settings descriptions for standalone agent also need to be updated? We have queue memory settings described for [Elasticsearch](https://www.elastic.co/guide/en/fleet/current/elasticsearch-output.html#output-elasticsearch-memory-queue-settings), [Logstash](https://www.elastic.co/guide/en/fleet/current/logstash-output.html#output-logstash-memory-queue-settings), and Kafka (currently in an open PR).

Those pages all show an example configuration like this:

```
queue.mem:
  events: 4096
  flush.min_events: 512
  flush.timeout: 5s
```

Reply:
The "expanded" YAML syntax is equivalent, but perhaps we should be consistent, e.g. 

```yaml
queue.mem.events: 4096
queue.mem.flush.min_events: 512
queue.mem.flush.timeout: 5s
```

is equivalent to 

```yaml
queue:
  mem:
    events: 4096
    flush:
      min_events: 512
      timeout: 5s
```

which is also equivalent to 

```yaml
queue.mem:
  events: 4096
  flush.min_events: 512
  flush.timeout: 5s
```

In YAML, dot notation or indent notation are both valid. It's largely subjective where one uses dots and where one introduces a new level of indentation. I'm in favor of aligning these example code blocks with what's in those other preexisting docs if only because it'd be less work than updating those other docs to match the way we have it in these ingest docs 🙂

Reply:
Brilliiant! Thanks @kpollich for the super clear explanation. I wasn't sure and certainly didn't want to just guess. :-) 

I've updated all instances of the `events`, `flush.min_events`, and `flush.timeout` settings to be `queue.mem.events`, `queue.mem.flush.min_events`, and `queue.mem.flush.timeout`, respectively.

The configuration examples in the standalone agent output settings docs are now like this:

```
  queue.mem.events: 4096
  queue.mem.flush.min_events: 512
  queue.mem.flush.timeout: 5s
```

The docs preview linked in the description will update once the CI checks have re-run.